samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties

```diff
@@ -361,6 +361,10 @@ access_denied=Access Denied
 access_denied_message=You do not have permission to access this assessment. Please contact your instructor if you have any questions.
 secure_delivery_error_take_message=An error occurred while retrieving the link to take this assessment through {0}. Please contact your instructor.
 secure_delivery_error_review_message=An error occurred while retrieving the link to review this assessment through {0}.
+seb_download_configuration=Download Configuration
```

Comment:
Same as above, create the strings in _es, _CA and _EU, and please, put all the strings at the end of the file into its own section.